server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/git/PathPatternFilter.java

```diff
@@ -226,6 +226,11 @@ public boolean matches(String path) {
             return true;
         }
 
+        if (path.charAt(0) == '/') {
+            // Strip the leading '/' since pathPatterns was compiled without it.
+            path = path.substring(1);
```

Comment:
Was this a bug?

Reply:
No, it isn't. The path of Central Dogma repository starts with `/` whereas the path of jGit does not start with `/`.
Previously, `PathPatternFilter` was designed to match the paths in jGit's tree. I changed it so that we can also use `PathPatternFilter` to filter `Entry.path()`.

Reply:
Thanks for the explanation. 👍